woocommerce/google-listings-and-ads
# Add Size, Size System, and Size Type product attributes

### Changes proposed in this Pull Request:



Part of #16 .

This PR adds Size, Size System, and Size Type product attributes. These attributes are available for simple products and variations (for variable products, each variation has its own input for these attributes).

### Detailed test instructions:

1. Create a new product or edit an existing one
2. If it's a simple product: change the values of Size, Size System, and Size Type in the "Google Listings and Ads" tab in the product data section
3. If it's a variable product: change the values of Size, Size System, and Size Type for each variation in the "Google Listings and Ads" section under each variation
4. Save the product and make sure it's synced to Google
5. Make sure Size, Size System, and Size Type are correctly updated in Google Merchant Center


### Changelog Note:

> Tweak - Add Size, Size System, and Size Type product attributes
